packages/toolpad-app/src/toolpad/Home.tsx

```diff
@@ -48,18 +48,41 @@ import useLocalStorageState from '../utils/useLocalStorageState';
 import ErrorAlert from './AppEditor/PageEditor/ErrorAlert';
 import { ConfirmDialog } from '../components/SystemDialogs';
 import config from '../config';
+import { AppTemplateId } from '../types';
 import { errorFrom } from '../utils/errors';
 
+export const APP_TEMPLATE_OPTIONS = {
```

Comment:
It would probably make sense to load the templates list from the server as well

Reply:
this a different thing though, it's a list with the texts we show in the UI, so meant for the browser-side only

Reply:
could be more type safe though if it's not, i thought i had made sure the key names matched the templates
i'll take another look at that

Reply:
Also, prefer to use a `Map<string, TemplateDef>` for these sort of things. Just avoids issues when someone decides to create a `'toString'` template or something.